automake/pkg.yaml

```diff
@@ -28,4 +29,4 @@ steps:
         make DESTDIR=/rootfs install
 finalize:
   - from: /rootfs
-  - to: /
```

Comment:
oh 🤦

Reply:
yeah that's what caused sources to be placed into the image, incl. bin from sources overwriting /bin